runner/app/pipelines/image_to_video.py

```diff
@@ -41,7 +43,28 @@ def __init__(self, model_id: str):
             kwargs["torch_dtype"] = torch.float16
             kwargs["variant"] = "fp16"
 
-        self.ldm = StableVideoDiffusionPipeline.from_pretrained(model_id, **kwargs)
+        try:
+            if any(substring in model_id.lower() for substring in ("ltx-video", "ltx")):
+                logger.info("Loading LTXImageToVideoPipeline for model_id: %s", model_id)
+                self.pipeline_name = "LTXImageToVideoPipeline"
+                self.ldm = LTXImageToVideoPipeline.from_pretrained(model_id, **kwargs)
+            else:
+                logger.info("Loading StableVideoDiffusionPipeline for model_id: %s", model_id)
+                self.pipeline_name = "StableVideoDiffusionPipeline"
+                self.ldm = StableVideoDiffusionPipeline.from_pretrained(model_id, **kwargs)
+        except Exception as loading_error:
+            logger.error("Failed to load %s : %s." %(self.pipeline_name,loading_error))
+            # Trying to load the LTXImageToVideoPipeline if the StableVideoDiffusionPipeline fails to load and there is a chance that model name doesn't match the if condition for LTX-Video
+            # (for future extra models support)
+            try:
+                logger.info("Trying LTXImageToVideoPipeline for model_id: %s", model_id)
+                self.pipeline_name = "LTXImageToVideoPipeline"
+                self.ldm = LTXImageToVideoPipeline.from_pretrained(model_id, **kwargs)
+            except Exception as loading_error:
+                logger.error("Failed to load both LTXImageToVideoPipeline and StableVideoDiffusionPipeline: %s. Please ensure the model ID is compatible.", loading_error)
+                raise loading_error
+            
+
         self.ldm.to(get_torch_device())
 
         sfast_enabled = os.getenv("SFAST", "").strip().lower() == "true"
```

Comment:
Need to confirm if SFAST works on LTXImageToVideoPipeline. If it does not, only use it if `StableVideoDiffusionPipeline` is loaded.

Reply:
Thanks for confirming